siriuspy/siriuspy/psdiag/csdev.py

```diff
@@ -12,15 +12,15 @@ class ETypes(_csdev.ETypes):
         'Current-(SP|Mon) differ', 'Interlocks', 'Wfm error')
 
 
-_et = ETypes  # syntactic sugar
+_et = ETypes  # syntatic sugar
 
 
 def get_ps_diag_propty_database(psname):
     """Return property database of diagnostics for power supplies."""
     pstype = _PSSearch.conv_psname_2_pstype(psname)
     splims = _PSSearch.conv_pstype_2_splims(pstype)
     dtol = splims['DTOL_CUR']
-    dbase = {
```

Comment:
What do think of using _dbase_ name in these situations? In the previous PR we replaced _db_ by _dbase_ in various locations in order to comply with pylint three-lettered name rule.

Reply:
see previous comment